mobile-app/app/screens/AppNavigator/screens/Dex/DexNavigator.tsx

```diff
@@ -100,14 +106,25 @@ export function DexNavigator (): JSX.Element {
       <DexStack.Screen
         component={AddLiquidityScreen}
         name='AddLiquidity'
-        options={{
+        options={({ navigation, route }: StackScreenProps<DexParamList, 'AddLiquidity'>) => ({
```

Comment:
Instead of customizing through props, we can add the screen into `BalancesNavigator`

Reply:
this screen too `ConfirmAddLiquidityScreen`

Reply:
Hmm, back of `ConfirmAddLiquidityScreen` should be to `AddLiquidityScreen`

Reply:
if you enter through Balance > Token Detail > Add liquidity > Confirm Add Liquidity
what would happen if you go back from Confirm Add Liquidity, will it still reach Token Detail screen?